header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -449,6 +451,69 @@ public function define_row_settings( $row_id ) {
 				'default'               => $default_colors['text'],
 			]
 		);
+
+		SettingsManager::get_instance()->add(
+			[
+				'id'                    => self::BOTTOM_BORDER,
+				'group'                 => $row_setting_id,
+				'tab'                   => SettingsManager::TAB_STYLE,
+				'section'               => $row_setting_id,
+				'label'                 => __( 'Divider width', 'neve' ),
+				'type'                  => '\Neve\Customizer\Controls\React\Responsive_Range',
+				'live_refresh_selector' => $row_class,
+				'live_refresh_css_prop' => [
+					'cssVar' => [
+						'responsive' => true,
+						'vars'       => '--headerBorderWidth',
+						'suffix'     => 'px',
+						'fallback'   => '0',
+						'selector'   => '.' . $this->get_id() . '-' . $row_id,
+					],
+					'prop'   => 'border-bottom-width',
+					'unit'   => 'px',
```

Comment:
These would be unnecessary as they address the way we were doing live-refresh in the old skin. 🏚️